Kernel/Storage/ATA/AHCIController.h

```diff
@@ -51,9 +50,8 @@ class AHCIController final : public ATAController
     NonnullOwnPtr<Memory::Region> default_hba_region() const;
     volatile AHCI::HBA& hba() const;
 
-    NonnullRefPtrVector<Memory::PhysicalPage> m_identify_metadata_pages;
-
     Array<RefPtr<AHCIPort>, 32> m_ports;
+
```

Comment:
This looks like a stray newline, or like something that should have gone into a different commit (as the surrounding lines actually got modified in an earlier commit).